lib/dirList.js

```diff
@@ -156,15 +157,12 @@ const dirList = {
    * say if the route can be handled by dir list or not
    * @param {string} route request route
    * @param {(boolean | ListOptionsJsonFormat | ListOptionsHtmlFormat)} options
-   * @return {boolean}
+   * @return {boolean | undefined}
    */
   handle: function (route, options) {
-    if (!options.names) {
-      return false
-    }
-    return options.names.includes(path.basename(route)) ||
+    return options.names?.includes(path.basename(route)) ||
       // match trailing slash
-      (options.names.includes('/') && route[route.length - 1] === '/')
+      (options.names?.includes('/') && route[route.length - 1] === '/')
```

Comment:
if we do
```suggestion
      (options.names?.includes('/') && route[route.length - 1] === '/') || false
```
then we dont return undefined and the api stays the same.

Reply:
Nit: even better would be to use `??` where false will _only_ be evaluated if the value is `undefined`